wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -198,15 +221,191 @@ def predict(self, X):
         return self.score(X).argmax(axis=1)
 
     def save(self, path):
+        """Saves model to path"""
         os.makedirs(path, exist_ok=True)
         self.model.save_pretrained(path)
 
     def load(self, path):
+        """Loads model from path"""
         self._initialise_models()
         self.model = TFBertForSequenceClassification.from_pretrained(path)
         self.trained_ = True
 
 
+class SemanticMetaBert(SemanticEquivalenceClassifier):
```

Comment:
To keep up with the convention of sklearn and for consistency maybe the class should be called SemanticEquivalenceMetaBertClassifier. I also think Bert is redundant in the name

Reply:
Agree - I changed this in https://github.com/wellcometrust/WellcomeML/pull/87/commits/36f5fdfc4f37ae3c79a415b96b903f964daf586f